core/src/test/java/com/datastrato/gravitino/storage/kv/TestKvGarbageCollector.java

```diff
@@ -234,4 +236,164 @@ void testRemoveWithGCCollector() throws IOException, InterruptedException {
       }
     }
   }
+
+  @Test
+  void testRemoveWithGCCollector2() throws IOException, InterruptedException {
```

Comment:
What's the difference between `testRemoveWithGCCollector2` and the old one?

Reply:
The unit test `testRemoveWithGCCollector2` was originally saved in the `TestKvEntityStorage` class. Since it has the same name as the unit test method in `TestKvGarbageCollector`, `1` and `2` are added to distinguish them. The test logic of `testRemoveWithGCCollector1` and `testRemoveWithGCCollector2` is slightly different, and there is no direct deletion here.